packages/sidenav/src/SideNav.story.js

```diff
@@ -0,0 +1,413 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { colors } from "@crave/farmblocks-theme";
+import Text from "@crave/farmblocks-text";
+import { Switch, Route, NavLink, withRouter } from "react-router-dom";
+import StoryRouter from "storybook-react-router";
+import { useMediaQuery } from "react-responsive";
+
+import SideNav from "./SideNav";
+import { FULLSCREEN, PUSH, OVERLAY } from "./constants/variants";
+import NavItem from "./components/NavItem";
+import PageWrapper from "./helpers/PageWrapper";
+import useToggle from "./utils/useToggle";
+
+const tabs = ["purveyor", "order", "search", "meat"];
+
+export default { title: "SideNav/SideNav", component: SideNav };
+
+export const SideNavSimple = () => (
+  <SideNav render={() => <div>Sidebar Content</div>} />
+);
+
+export const CompleteSideNavSimple = () => (
+  <>
+    <SideNav
+      render={props => (
+        <>
+          <NavItem active {...props}>
+            Item 1 (active)
+          </NavItem>
+          <NavItem {...props}>Item 2</NavItem>
+          <NavItem {...props}>Item 3</NavItem>
+        </>
+      )}
+    />
+    <PageWrapper expanded>
+      <h1>Page Content</h1>
+    </PageWrapper>
+  </>
+);
+
+const SideNavPushComp = () => {
+  const [expanded, { toggle, collapse }] = useToggle(false);
+  const [selected, setSelected] = useState(tabs[0]);
+
+  return (
+    <>
+      <SideNav
+        expanded={expanded}
+        onToggle={toggle}
+        onClose={collapse}
+        render={props => (
+          <>
+            <NavHeader />
+            {tabs.map(tab => (
+              <NavItem
+                key={tab}
+                onClick={() => setSelected(tab)}
+                active={tab === selected}
+                icon={`wg-${tab}`}
+                {...props}
+              >
+                {tab}
+              </NavItem>
+            ))}
+          </>
+        )}
+      />
+
+      <PageWrapper expanded={expanded}>
+        <LoremBlock variant={PUSH} />
+      </PageWrapper>
+    </>
+  );
+};
+
+export const SideNavPush = () => <SideNavPushComp />;
```

Comment:
Create the component inside the story function. This is not a good practice in React, but for storybook, it will keep the code visible in the docs.

Outside: 
![image](https://user-images.githubusercontent.com/1459283/68041891-5f033480-fcb0-11e9-817a-649b8db259a4.png)

Inside:
![image](https://user-images.githubusercontent.com/1459283/68041954-80fcb700-fcb0-11e9-969c-64828849510e.png)


The same goes to some other stories bellow